concent_api/conductor/tests/test_integration_conductor.py

```diff
@@ -210,6 +210,53 @@ def test_conductor_should_not_schedule_verification_order_task_if_it_was_already
             self.assertEqual(UploadReport.objects.count(), 3)
             self.assertEqual(mock_task.call_count, 1)
 
+    def test_conductor_should_schedule_verification_order_task_if_it_uploaded_file_path_was_not_existing_before(self):
```

Comment:
sth's wrong with the test name. Did you mean:
`test_conductor_should_not_schedule_upload_finished_if_uploaded_file_path_was_existing_before`

Reply:
:+1: done